src/syntax/mod.rs

```diff
@@ -16,21 +17,33 @@ pub struct File {
 }
 
 /// A source file's line with its location
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Eq, PartialEq)]
 pub struct Line {
     pub line: String,
     pub no: u32,
 }
 
 /// A `.` import
-#[derive(Debug)]
+#[derive(Debug, Clone, Eq, PartialEq)]
 pub struct Import {
+    pub resolved_import: Option<PathBuf>,
     pub location: Line,
     pub importee: Importee,
 }
+impl Ord for Import {
+    fn cmp(&self, other: &Import) -> Ordering {
+        self.location.no.cmp(&other.location.no)
+    }
+}
+impl PartialOrd for Import {
+    fn partial_cmp(&self, other: &Import) -> Option<Ordering> {
+        Some(self.cmp(other))
+    }
+}
```

Comment:
If we treat the `Import` as a "syntactic element in the source code", ordering on just one field would be surprising. Perhaps it'd be nice to differentiate between "Import as a syntax" element and "Import as a path" (for the latter, I'd just use bare Path)